bin/check_samplesheet.py

```diff
@@ -142,6 +142,7 @@ def check_samplesheet(file_in, file_out):
                     )
 
                 for idx, val in enumerate(sample_mapping_dict[sample]):
+                    ## TODO: Why is this code commented out?
```

Comment:
This code is waiting for v2.1 where we want to include controls

Reply:
Ok. I have removed it for in the upcoming updates. We can add that in later if required.